packages/theme/src/Components/Checkbox/index.tsx

```diff
@@ -0,0 +1,10 @@
+import type { FC } from 'react'
+import { Icons } from '@masknet/icons'
+import type { RadioProps } from '../Radio'
+
+export interface CheckboxProps extends RadioProps {}
+
+export const Checkbox: FC<CheckboxProps> = ({ checked, ...rest }) => {
```

Comment:
Please consider using https://react-spectrum.adobe.com/react-aria/useCheckbox.html for better accessibility. e.g. Being able to handle keyboard events.

Reply:
I think @nuanyang233 could take care of this.

Reply:
As with Radio, you can use mui's <Checkbox />

Reply:
OK